ui/src/app/components/runtime-configuration-panel/customize-panel.tsx

```diff
@@ -239,6 +242,35 @@ export const CustomizePanel = ({
               )}
             </FlexRow>
           </FlexColumn>
+
+          {analysisConfig.computeType === ComputeType.Standard &&
+            gceVmZones &&
+            gceVmZones.length > 1 && (
+              <FlexColumn>
+                <label style={styles.label} htmlFor='runtime-compute'>
+                  Zone
+                </label>
+                <TooltipTrigger
+                  content={`Cannot change the zone when an environment or persistent disk exists. 
```

Comment:
optional: we can use standard quotes, and also remove the brackets.

`content='whatever'`

Reply:
I believe that the curly braces are necessary, because content is expecting a ReactNode. The back ticks are to make line wrapping easier, so the linter does not complain about the length of the line.

Reply:
ok - I missed that this was multi-line